Doc/library/calendar.rst

```diff
@@ -170,6 +170,64 @@ it's the base calendar for all computations.
       sheet should be used. *encoding* specifies the encoding to be used for the
       output (defaulting to the system default encoding).
 
+   :class:`HTMLCalendar` has the following attributes you can override to
+   customize the style of your calender:
+
+   .. attribute:: cssclasses
+
+      A list of CSS styles used for each weekday. The default style list is::
+
+         cssclasses = ["mon", "tue", "wed", "thu", "fri", "sat", "sun"]
+
+      But you can add more styles for each day if you want::
+
+         cssclasses = ["mon text-bold", "tue", "wed", "thu", "fri", "sat", "sun red"]
+
+      Note that the length of this list must be 7 items.
+
+   .. attribute:: noday_classes
+
+      The CSS class for a week day occurring in previous or the coming month.
+
+   .. attribute:: weekday_head_classes
+
+      A list of CSS styles used for each weekday. The default style list is
+      the same as ``HTMLCalendar.cssclasses``.
```

Comment:
If you change this to :attr:`HTMLCalendar.cssclasses`, I think Sphinx will make a link.